d2l-input-shared-styles.html

```diff
@@ -36,7 +36,9 @@
 					font-size: 0.8rem;
 					font-weight: 400;
 					letter-spacing: 0.02rem;
-					line-height: var(--d2l-input-line-height);
+					/* using min-height instead of line-height as IE11
+					 * doesn't support line-height on inputs */
+					min-height: calc(2rem + 2px);
```

Comment:
Looks like we can just remove `--d2l-input-line-height` now, it's not used anywhere else

Reply:
Seems reasonable.  Presumably this results in the cursor extending to the min-height?  Just wondering if there are any side-effects (in other browsers) by removing the line-height 1.2rem that I think may be slightly different from browser default.  Open-dyslexic font also comes to mind since it's kinda weird, but I _think_ those issues cropped up when overflow was hidden.

Reply:
Thanks, I intended to do that after testing and then forgot. Done.

Reply:
I found absolutely no differences with the line height gone. I think the reason is that it now falls back to `line-height: normal`, which [the docs say](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) is roughly `1.2` -- so the same.